web/docs/auth/entities/entities.md

```diff
@@ -1,22 +1,454 @@
+  // ...
+}
+```
+
+```ts title=src/tasks.ts
+export const createTask: CreateTask<...>  = async (args, context) => {
+  const userId = context.user.getFirstProviderUserId()
+  // ...
+}
+```
+
+</TabItem>
+</Tabs>
+
+## Including the User with Other Entities
+
+Sometimes, you might want to include the user's data when fetching other entities. For example, you might want to include the user's data with the tasks they have created.
+
+:::caution Be careful to exclude the password
+
+You'll need to include the `auth` and the `identities` relations to get the full data about the user. However, you should exclude the `providerData` field from the `AuthIdentity` entity because it contains the user's hashed password.
+
+:::
+
+You can include the user's data with other entities using the `include` option in the Prisma queries:
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```js title="src/tasks.js"
+export const getAllTasks = async (args, context) => {
+  return context.entities.Task.findMany({
+    orderBy: { id: 'desc' },
+    select: {
+      id: true,
+      title: true,
+      // highlight-start
+      user: {
+        include: {
+          auth: {
+            include: {
+              identities: {
+                // Making sure to exclude the `providerData` field
+                select: {
+                  providerName: true,
+                  providerUserId: true,
+                },
+              },
+            },
+          },
+        },
+      },
+      // highlight-end
+    },
+  })
+}
+```
+
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```ts title="src/tasks.ts"
+export const getAllTasks = (async (args, context) => {
+  return context.entities.Task.findMany({
+    orderBy: { id: 'desc' },
+    select: {
+      id: true,
+      title: true,
+      // highlight-start
+      user: {
+        include: {
+          auth: {
+            include: {
+              identities: {
+                // Making sure to exclude the `providerData` field
+                select: {
+                  providerName: true,
+                  providerUserId: true,
+                },
+              },
+            },
+          },
+        },
+      },
+      // highlight-end
+    },
+  })
+}) satisfies tasks.GetAllQuery<{}, {}>
+```
+
+</TabItem>
+</Tabs>
+
+If you have some **piece of the auth data that you want to access frequently** (for example the `username`), it's best to store it at the top level of the `User` entity.
+
+For example, save the `username` or `email` as a property on the `User` and you'll be able to access it without fetching the auth-related fields. We show an example in the [Defining Extra Fields on the User Entity](../overview.md#1-defining-extra-fields) section of the docs.
+
+### Getting the User's Data
+
+If you fetch the user data like this, you'll lose the benefits of the `AuthUser` object. You'll need to access the auth data by going through the `auth` and `identities` relations:
```

Comment:
What does "like this" stand for here? I would try to make it more precise.

Maybe "When you have `user` object with `user.auth.identities` field, it can be a bit tedious to obtain the auth data (like username or google id) from it.